lte/gateway/c/core/oai/tasks/async_grpc_service/grpc_async_service_task.c

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
please use magma copyright not OAI.
Also you should code this thing in c++. You can use `extern "C" status_code_e grpc_async_service_init()`